### ISSUE TITLE
undefined method 'group_name' for nil:NilClass with security_group

### ISSUE THREAD

**Issue by craigmonson** [k1LoW/awspec#321]
Version: **0.85.2**

I generated a series of tests using the generate command.  They all worked, with the exception of those that are commented out below (note that one 'all' test that's passing for sg-22222222)

```ruby
describe security_group('SG_TestTest02') do
  it { should exist }
  its(:group_id) { should eq 'sg-11111111' }
  its(:group_name) { should eq 'SG_TestTest02' }
  its(:inbound) { should be_opened(1433).protocol('tcp').for('x.x.x.x/32') }
  its(:inbound) { should be_opened(80).protocol('tcp').for('x.x.x.x/16') }
  its(:inbound) { should be_opened(80).protocol('tcp').for('x.x.x.x/32') }
  its(:inbound) { should be_opened.protocol('all').for('x.x.x.x/24') }
  its(:inbound) { should be_opened.protocol('all').for('x.x.x.x/24') }
  # these indeed exist, but there seems to be a bug with awspec at least at version 0.85.2
#  its(:inbound) { should be_opened.protocol('all').for('sg-33333333') }
#  its(:inbound) { should be_opened.protocol('all').for('sg-44444444') }
  its(:inbound) { should be_opened.protocol('all').for('sg-22222222') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/16') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/16') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/20') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
#  its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }
  its(:outbound) { should be_opened.protocol('all').for('0.0.0.0/0') }
  its(:inbound_rule_count) { should eq 17 }
  its(:outbound_rule_count) { should eq 1 }
  its(:inbound_permissions_count) { should eq 4 }
  its(:outbound_permissions_count) { should eq 1 }
  it { should belong_to_vpc('vpc - vpc') }
end
```

This generates the same error for each failing test:
```
Failure/Error: its(:inbound) { should be_opened(443).protocol('tcp').for('x.x.x.x/32') }

      NoMethodError:
        undefined method `group_name' for nil:NilClass
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:106:in `block in cidr_opened?'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:98:in `select'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:98:in `cidr_opened?'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:26:in `block in inbound_opened?'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:25:in `each'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:25:in `find'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:25:in `inbound_opened?'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/type/security_group.rb:15:in `opened?'
      # /usr/local/bundle/gems/awspec-0.85.2/lib/awspec/matcher/be_opened.rb:3:in `block (2 levels) in <top (required)>'
      # ./spec/us-west-2_spec.rb:130:in `block (2 levels) in <top (required)>'
```

**Comment by k1LoW** [comment]
Hi @craigmonson ! Thank you for your report !
I'll check it.

**Comment by darrenhaken** [comment]
Was there a fix for this in the end?

**Comment by k1LoW** [comment]
Hi @craigmonson ! I just released v0.82.3! Could you check this version ?

**Comment by craigmonson** [comment]
@k1LoW  That seems to have fixed it!  Thanks!
